src/resolvers/exotics/file-resolver.js

```diff
@@ -23,6 +23,11 @@ export default class FileResolver extends ExoticResolver {
   loc: string;
 
   static protocol = 'file';
+  static prefixMatcher = /^.{,2}\//;
+
+  static isVersion(pattern: string): boolean {
+    return super.isVersion.call(this, pattern) || this.prefixMatcher.test(pattern);
```

Comment:
Maybe update the regex to also match `C:\...` ?

Reply:
Hmm, good catch. I'll go with [path.isAbsolute](https://nodejs.org/api/path.html#path_path_isabsolute_path) which takes care of it for me.